sound/hfe_planet_sound.asset

```diff
@@ -22,4 +22,56 @@ category = {
 	soundeffects = {
 		hfe_pc_memorial
 	}
+}
+
+sound = {
+	name = impossible_ahet_home_click
```

Comment:
Should you add mod prefix

Reply:
The last time I changed the sound, I broke it. I tested the code you see and it works. No changes I've made have allowed it to work.